pushnotifications/src/main/java/com/pusher/pushnotifications/reporting/FCMMessageReceiver.kt

```diff
@@ -23,6 +25,11 @@ class FCMMessageReceiver : WakefulBroadcastReceiver() {
   override fun onReceive(context: Context?, intent: Intent?) {
     Log.i("FCMMessageReceiver", intent.toString())
 
+    if (!FeatureFlagManager.isEnabled(FeatureFlag.DELIVERY_TRACKING)) {
+      Log.i("FCMMessageReceiver", "Delivery tracking flag is disabled. Skipping.")
```

Comment:
You can use the `log` instance you already have as part of the `FCMMessageReceiver` class and do:
`log.i("Delivery tracking flag is disabled. Skipping.")`